deep_qa/layers/tuple_matchers/slot_similarity_tuple_matcher.py

```diff
@@ -1,11 +1,11 @@
 from copy import deepcopy
-from typing import Any, Dict
+from typing import Dict, Any
```

Comment:
In fact, the whole file should stay how it was, except for the change from `get` to `pop`.